client/styles/base.scss

```diff
@@ -1,12 +1,18 @@
 @import 'palette';
 
-body {
-    font-family: "Roboto", "Helvetica", "sans-serif";
+body, label, button {
```

Comment:
`label` and `button` will inherit their font style from the body, so you shouldn't have to do this.  If there are specific styles you need to apply to labels and buttons, please put those elsewhere.